editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Publish/PublishFlowButton.tsx

```diff
@@ -147,7 +148,7 @@ export const PublishFlowButton = () => {
                 <Box pb={2}>
                   <Typography variant="body2">
                     {`Preview these content changes in-service before publishing `}
-                    <Link href={urls.preview} target="_blank">
+                    <Link href={previewURL} target="_blank">
```

Comment:
nit: should we be using `rel="noopener"` in combo with the `target="_blank"` - I've always done it for security reasons, but I don't know enough to determine whether it's actually useful in all scenarios

Reply:
Good spot! When linking internally this isn't required, but is [best practice when using any third-party resources](https://mui.com/material-ui/react-link/#security).